core/base-service/base-static.js

```diff
@@ -33,8 +34,12 @@ module.exports = class BaseStaticService extends BaseService {
         queryParams
       )
 
-      const badgeData = this._makeBadgeData(queryParams, serviceData)
-      // The final capture group is the extension.
+      const badgeData = coalesceBadge(
+        queryParams,
+        serviceData,
+        this.defaultBadgeData,
+        this
+      ) // The final capture group is the extension.
```

Comment:
The comment applies to the line below right (`const format = match.slice(-1)[0]`)? Might be better to have the comment on its own line instead of on the closing parenthesis.

Reply:
Yes, go for it… though tbh I doubt I put that there so it probably was Prettier. If it doesn’t stay where you want you could try adding a blank line.